### ISSUE TITLE
www.flightradar24.com no longer working in Waterfox Classic

### ISSUE THREAD

**Issue by robbely** [MrAlex94/Waterfox-Old#195]
Obscure site I know, but I think they made some changes as I remember some while back there was an invitation to "try the new site". Today the page no longer full loads - a header at the top of the page but no map or aircraft pins.

Any suggestions would be welcome.

**Comment by hawkeye116477** [comment]
Last update for that browser was at November, so I'd suggest to migrate to another one. If you need XUL extensions, then Pale Moon or Basilisk, otherwise you can move to Waterfox G.x.

**Comment by robbely** [comment]
> Last update for that browser was at November, so I'd suggest to migrate to another one. If you need XUL extensions, then Pale Moon or Basilisk, otherwise you can move to Waterfox G.x.

Yes, I'm aware of the alternatives, they've been posted many times here. I was looking for a solution that works with Waterfox Classic.

**Comment by hawkeye116477** [comment]
> for a solution that works with Waterfox Classic.

Unfortunately you can do nothing about that, cuz it would need to be fixed in Waterfox's code.

**Comment by tharlab** [comment]
> Last update for that browser was at November, so I'd suggest to migrate to another one. If you need XUL extensions, then Pale Moon or Basilisk, otherwise you can move to Waterfox G.x.

basilisk doesnot support webextension, but  thanks for mention it...
as replacement waterfox classic, 
whereas palemoon is broken doesnot work for me
any alternatif for waterfox classic that cover both world...
